Exercise 1/styles.css

```diff
@@ -25,3 +33,24 @@ footer {
     padding: 20px;
     border: #08313a 1px solid;
 }
+
+header {
+    grid-area: header;
+}
+
+nav {
+    grid-area: nav;
```

Comment:
Personally I wouldn't abbreviate this. While this might be a clear abbreviation, you'll and up using abbreviations that aren't clear to everyone.